co2calculator/distances.py

```diff
@@ -287,36 +299,6 @@ def geocoding_train_stations(loc_dict):
     return res_station_name, res_country, coords
 
 
-def is_valid_geocoding_dict(geocoding_dict):
-    """Function to check if the dictionary is valid as input for pelias structured geocoding. Raises error if it is not
-    the case
-
-    :param geocoding_dict: dictionary describing the location
-    """
-    allowed_keys = [
-        "country",
-        "region",
-        "county",
-        "locality",
-        "borough",
-        "address",
-        "postalcode",
-        "neighbourhood",
-    ]
-    assert len(geocoding_dict) != 0, "Error! Empty dictionary provided."
-    for key in geocoding_dict:
-        assert (
-            key in allowed_keys
-        ), f"Error! Parameter {key} is not available. Please check the input data."
-    # warnings
-    if "country" not in geocoding_dict.keys():
-        warnings.warn("Country was not provided. The results may be wrong.")
-    if "locality" not in geocoding_dict.keys():
-        warnings.warn(
-            "Locality (city) was not provided. The results may be inaccurate."
-        )
-
-
 def get_route(coords: list, profile: str = None) -> Kilometer:
```

Comment:
Cool, didn't know that type hint `Kilometer` was possible. :)

Reply:
Yea, it is, because we defined it here: https://github.com/pledge4future/co2calculator/blob/ff86e53720e6b096356601285789982977cbfd69/co2calculator/_types.py#L4-L7